agent/src/agent/pipeline/config/stages/source/elastic.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from agent import source, monitoring
+from agent.pipeline.config.stages.base import JythonSource
+
+
+class ElasticScript(JythonSource):
+    JYTHON_SCRIPT = 'elastic_loader.py'
+
+    def _load_query_from_file(self):
+        with open(self.pipeline.config['query_file']) as f:
+            query = f.read()
+            return query.replace('${OFFSET}', '"OFFSET"')
+
+    def _get_script_params(self) -> list[dict]:
+        return [
+            {
+                'key': 'URLs',
+                'value': self.pipeline.source.config['conf.httpUris']
+            },
+            {
+                'key': 'INDEX',
+                'value': self.pipeline.source.config['conf.index']
+            },
+            {
+                'key': 'INITIAL_TIMESTAMP',
+                'value': str(int(self.get_initial_timestamp().timestamp()))
+            },
+            {
+                'key': 'INTERVAL',
+                'value': str(self.pipeline.source.config.get('query_interval_sec', 1))
+            },
+            {
+                'key': 'DELAY_IN_MINUTES',
+                'value': str(self.pipeline.delay)
+            },
+            {
+                'key': 'QUERY_TIMEOUT',
+                'value': str(self.pipeline.source.query_timeout)
+            },
+            {
+                # Do we need to customize it?
+                'key': 'SCROLL_TIMEOUT',
+                'value': '5m'
```

Comment:
As I right understand, the `scroll` parameter (passed to the search request and to every scroll request) tells Elasticsearch how long it should keep the search context alive. Do we really need this?
To receive a bigger amount of hits, we could pass `"size": 10000` into the request payload, for example

Reply:
we should keep the scroll logic but it should handle it in the script, we don't want to limit requests by size and the response can be quite big and memory consuming